dev/tests/unit/Util/MagentoTestCase.php

```diff
@@ -19,6 +19,8 @@ public static function setUpBeforeClass(): void
         if (!self::fileExists(DOCS_OUTPUT_DIR)) {
             mkdir(DOCS_OUTPUT_DIR, 0755, true);
         }
+        // Should be used to clean AspectMock mocking before using PHPUnit mocking and Reflection.
+        AspectMock::clean();
```

Comment:
This Aspect Mock usage needs to be removed.

Reply:
Hello, @jilu1 
It is temporarily. When all PRs merged we will proceed with the AspectMock eliminating in this class.

Reply:
@bohdan-harniuk 
Got it. Thanks for the explanation!